### ISSUE TITLE
The project Id is displayed under Team Name column for some resources

### ISSUE THREAD

**Issue by sandhiyakavi** [topcoder-platform/micro-frontends-taas-admin-app#49]
Description:

Under the Team Name column, I am able to see projectID displaying instead of Team name for some resources.

![image](https://user-images.githubusercontent.com/55895538/123367035-40079380-d5b4-11eb-9403-5939f1fce5a3.png)

**Comment by maxceem** [comment]
@sandhiyakavi I cannot reproduce this issue. Do you know any steps on how to reproduce it? If you could reproduce it again, could you please check the DevTool console log and see what error do you see there.

PS. I suspect that it was some temporary issue with Projects API.

**Comment by sandhiyakavi** [comment]
@maxceem Yes, I am not able to reproduce the issue, The Team name's are correctly displaying now.

**Comment by sandhiyakavi** [comment]
@maxceem It is happening for the BookingManager role

![image](https://user-images.githubusercontent.com/55895538/124437314-e7a17480-ddb1-11eb-8b58-dc1b84c7c4ab.png)
